### PR TITLE
Slightly better gib

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Human gib animations are no longer 32x32
They now also vary based on ethnicity (other species like robots/moths already have support for species specific gib animations, but no one ever made any).

Someone else could do miles better, but at least its better than it currently is.

https://github.com/user-attachments/assets/222ca693-d2c4-42df-ad8e-f233b94182f4
## Why It's Good For The Game
Looks a bit less jarring compared to when your unga gets gibbed, and turns into a bright orange 2008 bay era sprite. Also making it larger than 32x32 does wonders to make it less obviously terrible.
## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. 
:cl:
imageadd: Improved human gib animations slightly
/:cl:

### CODE REVIEW

**Comment by Xander3359** [comment]
holy shit it looks brutal as hell

**Review by Kuro020 — APPROVED** [review]

File: code/game/objects/effects/overlays.dm [hunk 1]
```diff
@@ -331,36 +331,35 @@
...
 /obj/effect/overlay/temp/gib_animation/xeno
 	icon = 'icons/Xeno/64x64_Xeno_overlays.dmi'
 	effect_duration = 10
 
 /obj/effect/overlay/temp/gib_animation/xeno/Initialize(mapload, effect_duration, mob/source_mob, gib_icon, new_icon)
-	. = ..()
 	icon = new_icon
-
-
+	return ..()
```
Comment (QualityVan) [inline code/game/objects/effects/overlays.dm:362]: Is this for something in parent init needing icon set properly? Setting parent vars before parent init feels weird.

Comment (Lumipharon) [reply code/game/objects/effects/overlays.dm:362]: It's needed for the flick to use the right icon

**Comment by yyzsong** [comment]
I want my gibbing animation to be jarring. That's kind of the point of getting gibbed by a huge fuckoff explosion

**Review by QualityVan — APPROVED** [review]

**Comment by yyzsong** [comment]
it's soulver

**Pull request merged**